examples/storm-surge/ike/setplot.py

```diff
@@ -161,10 +162,10 @@ def gulf_after_axes(cd):
     plotaxes.ylimits = gulf_ylimits
     plotaxes.afteraxes = gulf_after_axes
 
-    surge.add_surface_elevation(plotaxes, plot_type='contourf', 
+    surge.plot.add_surface_elevation(plotaxes, plot_type='contourf',
```

Comment:
At some point there was a PR that replaced these updates and I have not had a chance to go back through and sync everything up.  Let me know if you find any other issues with the data objects as I imagine there are more.